src/components/RecipesList.js

```diff
@@ -26,6 +28,11 @@ class RecipesList extends React.Component {
     this.setState({ing: ""})
   };
 
+  onDeleteIngredient(index) {
+    this.state.ingredients.splice(index, 1)
```

Comment:
Should this state be altered directly?

Reply:
I see your point but I honestly do not see the difference if it is wrapped into a constant

Reply:
I think the difference is that React will 'see' the state change if you use their setState function. I know it kicks up warnings about altering state directly and not through setState. Could be why it wasn't re rendering right away and you have to call setState right afterwards. I think at this stage we should move on and come back around to it.

Reply:
@l-palermo - what do you mean wrapped in a constant?